example/lib/main.dart

```diff
@@ -109,6 +109,11 @@ class MyAppState extends State<MyApp> {
                   config: Config(
                     height: 256,
                     checkPlatformCompatibility: true,
+                    itemsOrderConfig: const ItemsOrderConfig(
+                      top: EmojiPickerItem.categoryBar,
+                      middle: EmojiPickerItem.emojiView,
+                      bottom: EmojiPickerItem.searchBar,
+                    ),
```

Comment:
Nit: Might would use the term `view` instead of `item`, what do you think?

Reply:
done